salt/modules/nftables.py

```diff
@@ -355,9 +399,48 @@ def get_rules(family="ipv4"):
     return rules
 
 
+def get_rules_json(family="ipv4"):
+    """
+    .. versionadded:: Sodium
```

Comment:
looks like this will need to be updated since it did not make it in sodium

Reply:
Can I replace it with `INSERTSOMEFUTUREVERSIONHERE` so I won't have to come back and do it again? 😆

Reply:
thats actually a good idea TBH. Every release we search/replace <release name> for <number version> anyways. It wouldn't be such a bad idea to create a standard NEXTRELEASE notation that we could use. Let me propose the idea to the team in a PR.

Reply:
https://github.com/saltstack/salt/pull/58154